src/Uno.UI/UI/Xaml/Media/Transform.cs

```diff
@@ -88,7 +76,9 @@ internal Matrix3x2 ToMatrix(Point relativeOrigin, Size viewSize)
 		// Currently we support only one view par transform.
 		// But we can declare a Transform as a static resource and use it on multiple views.
 		// Note: This is now used only for animations
-		internal virtual _View View { get; set; }
+		internal virtual object Owner { get; set; }
```

Comment:
Please don't : If I'm right, a `Transform` has no `Owner` as it can be used on multiple elements (especially when used as static resources as mentioned above). We are looking to remove that `View` property, don't build new feature on it.